src/docs/tutorial-transaction-process/create-transaction-process/index.md

```diff
@@ -0,0 +1,411 @@
+---
+title: Create a new transaction process
+slug: create-transaction-process
+updated: 2020-07-22
+category: tutorial-transaction-process
+ingress:
+  This guide describes how to create a new transaction process and how
+  to take it into use on the FTW-daily template.
+published: true
+---
+
+## Create a transaction process
+
+In this tutorial, we'll create a new transaction process for the
+CottageDays marketplace. It will be a daily booking process in contrast
+to the default nightly process. We make this for demonstration purpose
+since it might not be a smart move for an actual cottage-rental
+marketplace.
+
+In addition, we also add a new decline transition for the operator and
+update the client app so that it uses the new process.
+
+### Clone Flex example processes repository
+
+Writing _process.edn_ file and email templates from scratch is a bit
+tedious task. We'll make our life a bit easier by cloning Flex example
+processes repository:
```

Comment:
I think we could mention more clearly already here which process we are going to use as a starting point. Then we could move the extra info about preauth-daily-booking process already to this section.

Maybe we could even add a link to the example process repo and explain briefly what kind of processes there are. Though most of them are using deprecated pricing actions so maybe that's not very useful after all.